### ISSUE TITLE
Customer reported contrast fail on CTA button hover state in the ACP upsell on our settings page 

### ISSUE THREAD

**Issue by christophermhinds** [equalizedigital/accessibility-checker#694]
I did not follow the standard bug report format because I believe this screenshot should be sufficient.

<img width="292" alt="Screenshot 2024-06-28 at 8 43 37 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/25856499/0c53d37e-22fc-4ce2-9203-c57cb97d79a5">

Happy to re-submit if necessary.

**Comment by pattonwebz** [comment]
@christophermhinds can you ask this customer what browser and version of it they are using? I see something different when I spin up a fresh site for this button. It looks like what they are seeing is backwards compared to what I see.

https://github.com/equalizedigital/accessibility-checker/assets/3902039/e4bfd6f9-4a09-46a3-b38a-0ef7cad37e7d

**Comment by tdrayson** [comment]
Hey @pattonwebz, I'm using FireFox (developer edition), version 128.

I found removing `z-index: 1;` from the `.edac-pro-callout-button` https://github.com/equalizedigital/accessibility-checker/blob/8b602e744374add199c79bb7c00c3c2ba6e70d7b/src/admin/sass/accessibility-checker-admin.scss#L1025

and adding `z-index: 0` to `edac-pro-callout` https://github.com/equalizedigital/accessibility-checker/blob/8b602e744374add199c79bb7c00c3c2ba6e70d7b/src/admin/sass/accessibility-checker-admin.scss#L973

This seems to fix the issue.

**Comment by pattonwebz** [comment]
@tdrayson thanks for sharing the browser version you are using. When I tested with FF I found a similar fix worked as well but to figure out why I dug a little deeper and discovered that the difference between what is seen in FF and what I seen in my Chrome when I first looked comes down to differences in how they handle stacking contexts.

Firefox appears to be the only browser that I tested that implements stacking contexts exactly to specification and other browsers have a more loose implementation.

I'll have a solution pushed up tomorrow, will be released in the next plugin release.

Appreciate you reporting this and bringing it to our attention, this was interesting for me to dig into.

**Comment by tdrayson** [comment]
Perfect, thanks for the investigation. Always fun when different browsers implement specifications differently.... 🤣
